templates/webTemplateFiles.yaml

```diff
@@ -126,42 +126,42 @@
   description: Provides templates for miscellaneous use cases
 - lang: go
   id: xylo.numbers
```

Comment:
Rename the IDs.

Reply:
Yeah, I created the same review some days ago but forgot to submit it. :(
@philipgriggs Can you change the IDs?  Maybe to something like philipgriggs.numbers or shorter.